data/json/species.json

```diff
@@ -60,6 +60,13 @@
     "description": "a nether creature",
     "flags": [ "DRACULIN_IMMUNE" ]
   },
+  {
+    "type": "SPECIES",
+    "id": "NETHER_BURROWING",
+    "description": "a nether creature",
+    "footsteps": "slithering",
```

Comment:
I wouldn't go with slithering. I guess it'd be rumbling or rustling or whatever for burrowing, but slithering isn't the noise I'd expect to hear.

Reply:
Thank you! I couldn't think of any words other than "vibrations" and I was like that's gonna just be weird.